File: iFlat/src/main/java/com/iflat/system/entity/UserRoleVo.java
package com.iflat.system.entity;

/**
 * Created by tyriv on 2015/9/30.
 */
public class UserRoleVo {

    private String roleId;

    private String roleName;

    private String account;

    private String userName;

    private String category;

    private int sequence;

    public String getRoleId() {
        return roleId;
    }

    public void setRoleId(String roleId) {
        this.roleId = roleId;
    }

    public String getRoleName() {
        return roleName;
    }

    public void setRoleName(String roleName) {
        this.roleName = roleName;
    }

    public String getAccount() {
        return account;
    }

    public void setAccount(String account) {
        this.account = account;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public int getSequence() {
        return sequence;
    }

    public void setSequence(int sequence) {
        this.sequence = sequence;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }
}

File: web/app/model/system/UserRoleVo.js
Ext.define('iFlat.model.system.UserRoleVo', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'userRoleVo.roleId', mapping: 'roleId', type: 'string'},
        {name: 'userRoleVo.roleName', mapping: 'roleName', type: 'string'},
        {name: 'userRoleVo.account', mapping: 'account', type: 'string'},
        {name: 'userRoleVo.userName', mapping: 'userName', type: 'string'},
        {name: 'userRoleVo.category', mapping: 'category', type: 'string'}
    ]
});